### ISSUE TITLE
App groups not getting deployed

### ISSUE THREAD

**Issue by arun-gupta** [dcos-labs/dcos-maven-plugin#11]
App definition at https://github.com/arun-gupta/docker-java-sample/blob/master/app-definition.json can be deployed using `dcos marathon group add app-definition.json`. 

But the plugin gives the following error:

```
Response from DC/OS [422] {"message":"Object is not valid","details":[{"path":"/","errors":["AppDefinition must either contain one of 'cmd' or 'args', and/or a 'container'."]}]}
```

App and instructions: https://github.com/arun-gupta/docker-java-sample

**Comment by unterstein** [comment]
Hi,

please try using freshly released version `0.2` and configure your deployment as group `<deployable>group</deployable>`. I also updated the docs to cover groups and pods. Please let me know how.

Please let me know about your experiences :)

**Comment by arun-gupta** [comment]
App definition at https://github.com/arun-gupta/docker-java-sample/blob/master/app-definition.json works.

As discussed, `,deployable>group</deployable>` is more for the tool and not for the user. This should be optional in a future release.
